### PR TITLE
LPS-169972 Sort Assign Users and Invite Users menu items alphabetically

### PR DESCRIPTION
**Pull request by pei-jung**

### CODE REVIEW

**Comment by pei-jung** [comment]
Screen Shot 2022-11-30 at 2 17 46 PM

File: modules/apps/account/account-admin-web/types/src/main/resources/META-INF/resources/account_entries_admin/js/InviteUsersFormGroup.d.ts [hunk 1]
```diff
@@ -12,8 +12,6 @@
  * details.
  */
 
-/// <reference types="react" />
-
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/types/src/main/resources/META-INF/resources/account_entries_admin/js/InviteUsersFormGroup.d.ts:16]: Was this one intentional?

**Review by drewbrokke — APPROVED** [review]
Just the one comment, pretty unrelated. LGTM 👍

File: modules/apps/account/account-admin-web/types/src/main/resources/META-INF/resources/account_entries_admin/js/InviteUsersFormGroup.d.ts (hunk 1 above)
Comment (pei-jung) [reply modules/apps/account/account-admin-web/types/src/main/resources/META-INF/resources/account_entries_admin/js/InviteUsersFormGroup.d.ts:16]: Oops, that's unintentional. I'll remove it from the commit. Thanks for catching it!

**Comment by Sharryxue** [comment]
No related issue found in ci:test:relevant.

**Comment by drewbrokke** [comment]
ci:forward

**Review by drewbrokke — APPROVED** [review]

**Comment by drewbrokke** [comment]
ci:reevaluate:1267401_6187

**Comment by Sharryxue** [comment]
Will manually forward the commit.

**Comment by Sharryxue** [comment]
ci:close

**Pull request closed without merging**